docs/installing/adding_devices.md

```diff
@@ -91,14 +166,14 @@ Now that your edge device is prepared, continue on to [Installing the agent](reg
 * MicroSD flash card (32 GB preferred)
 * An appropriate power supply for your device (2 Amp or greater preferred)
 * An internet connection for your device (wired or wifi).
-  Note: Some devices require extra hardware for supporting wifi.
+  **Note**: Some devices require extra hardware for supporting wifi.
 * (optional) Sensor hardware: Many {{site.data.keyword.horizon}} edge services require specialized sensor hardware.
 
 ### Procedure
 {: #proc-pi}
 
 1. Prepare your Raspberry Pi device.
-   1. Flash the [Raspbian ](https://www.raspberrypi.org/downloads/raspbian/){:target="_blank"}{: .externalLink}{{site.data.keyword.linux_notm}} image onto your MicroSD card.
+   1. Flash the [Raspbian ](https://www.raspberrypi.org/downloads/raspbian/){:target="_blank"}{: .externalLink} {{site.data.keyword.linux_notm}} image onto your MicroSD card.
```

Comment:
It's now Raspberry Pi OS, not Raspbian.  We may need to do a blanket search-and-replace for the old term.

Reply:
I have opened https://github.com/open-horizon/open-horizon.github.io/issues/222 to track that work.